client/src/pages/login.tsx

```diff
@@ -61,25 +60,10 @@ export const Login: React.FC = () => {
         justifyContent="center"
         flexDirection="column"
       >
-        <ThemedTitleV2
-          collapsed={false}
-          wrapperStyles={{
-            fontSize: "22px",
-            justifyContent: "center",
-          }}
-        />
-
+        <AppName style={{
```

Comment:
You can change the name of the application only through Typography component, no need to introduce new component, even that it is a good idea, please make it simpler by using Typography.